src/content/tutorial/react-step-1/react-step-1.mdx

```diff
@@ -1,136 +1,169 @@
-Run `yarn` to install current dependencies.
+After the dependencies are installed, you can start the app with:
+
+```bash
+yarn start
+```
 
-#### Add Carbon dependencies
-Even though we installed existing dependencies, we're still missing a few Carbon packages we'll need.
+In a separate console, run `yarn test` to run the initial tests. When prompted for a watch mode, you can press `a` to run all tests when the app's source code is updated.
+
+You'll see failing tests. Don't worry - we'll make those tests pass as we build our app.
+
+## Install Carbon
+
+Even though we installed existing dependencies, we've yet to install the Carbon packages.
+
+`carbon-components` is the package that has our styles. We need to install this to be able to import our styles for our components.
 
-`carbon-components` is the package that has our styles. We will need to install this to be able to import our styles for our components.
 `carbon-components-react` is the package that has the React components that we will use throughout our app.
 
-And lastly `@carbon/icons-react` is part of our new [carbon-elements](https://github.com/carbon-design-system/carbon-elements) package. This collection of packages helps developers execute the IBM Design Language in products. There are several packages that we will use, but in the meantime we only need the icons package. 
+And lastly `@carbon/icons-react` is part of the [carbon-elements](https://github.com/carbon-design-system/carbon-elements) package. This collection of packages helps developers execute the IBM Design Language in products. There are several packages that we will use, but in the meantime we only need the icons package.
 
-```
+Stop your development server with `CTRL-C` and install Carbon dependencies with:
+
+```bash
 $ yarn add carbon-components carbon-components-react @carbon/icons-react
 ```
 
-Run `yarn start` to start your server. A browser window should automatically open with your local environment.
-```
+Then, start the app again.
+
+```bash
 $ yarn start
 ```
 
-## Setting up Sass
+## Install & build Sass
+
 Run the following command to install `node-sass` as a dependency.
-```
+
+```bash
 $ yarn add node-sass
 ```
 
 In `src/` rename `index.css` as `index.scss`. Then in `index.js` update the `css` import to `index.scss`.
 
 #### Importing carbon-component styles
+
 In `index.scss` import Carbon styles by adding:
-```
-@import 'carbon-components/scss/globals/scss/styles.scss';
 
+```scss
+@import 'carbon-components/scss/globals/scss/styles.scss';
 ```
 
 Next we'll import a `<Button/>` from Carbon to test that our dependencies are working properly. At the top of `src/App.js`, import the `<Button/>` by adding the following:
 
-```
+```javascript
 import Button from 'carbon-components-react/lib/components/Button/Button';
-``` 
+```
 
 We want to import the exact path of the component in this case. To minimize our import cost, we don't want to import the entire package just for the `Button` component.
```

Comment:
Looking back, I feel like this could be worded as:

_We want to import the exact path of the component instead of the entire package in order to minimize our import cost._

or 

_We want to minimize our import cost by avoiding an import of the entire package. To do this, we import the exact path of the component._

Reply:
Got it.